com.unity.shadergraph/Documentation~/Object-Node.md

```diff
@@ -16,6 +16,11 @@ Note: The behaviour of the Position [Port](Port.md) can be defined per Render Pi
 |:------------ |:-------------|:-----|:---|:---|
 | Position      | Output | Vector 3 | None | Object position in world space |
 | Scale       | Output | Vector 3 | None | Object scale in world space |
+| World Bounds Min | Output | Vector 3 | None | Minimum value of the renderer bounds in world space |
+| World Bounds Max | Output | Vector 3 | None | Maximum value of the renderer bounds in world space |
+| Bounds Size | Output | Vector 3 | None | Size of the renderer bounds |
+
+Note: the bounds values are the equivalent of [the bounds in the renderer component](https://docs.unity3d.com/ScriptReference/Renderer-bounds.html). This means that vertex deformation done in ShaderGraph don't affect these values.
```

Comment:
Should probably reiterate in documentation that this is not available for built-in.